app/helpers/mustache_helper.rb

```diff
@@ -64,13 +64,16 @@ def js_variables
 
   def js_files
     [
-      { path: asset_path('jquery.js') },
+      #{ path: asset_path('jquery.js') },
 
-      { path: 'http://develop.styleguide.eanadev.org/js/dist/global.js' },
-      { path: 'http://develop.styleguide.eanadev.org/js/dist/channels.js' },
+      #{ path: 'http://develop.styleguide.eanadev.org/js/dist/global.js' },
+      #{ path: 'http://develop.styleguide.eanadev.org/js/dist/channels.js' },
 
-      #{ path: 'http://localhost/x/Europeana-Patternlab/public/js/dist/channels.js' },
-      #{ path: 'http://localhost/x/Europeana-Patternlab/public/js/dist/global.js' },
+      #{ path: 'http://localhost/Europeana-Patternlab/public/js/dist/channels.js' },
+      #{ path: 'http://localhost/Europeana-Patternlab/public/js/dist/global.js' },
+      
+      #{ path: 'http://localhost/Europeana-Patternlab/public/js/dist/require.js',  data_main: 'http://localhost/Europeana-Patternlab/public/js/dist/main'},
+      { path: 'http://develop.styleguide.eanadev.org/js/dist/require.js',  data_main: 'http://develop.styleguide.eanadev.org/js/dist/main'},
```

Comment:
Line is too long. [140/80]<br>Unnecessary spacing detected.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.